label.rb

```diff
@@ -0,0 +1,24 @@
+class Label
+  attr_accessor :title, :color, :id
+
+  def initialize(title, color, _id)
+    @id = rand(0..100_000)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  attr_reader :items
+
+  def add_item(item)
+    @items.push(item) unless @items.include? item
+    item.label = self
+  end
+end
```

Comment:
Awesome. This is also going to close the  Implement methods. Nice work ✔ Closes #11